exercises/binary/binary_test.go

```diff
@@ -39,10 +39,14 @@ var testCases = []struct {
 }
 
 func TestParseBinary(t *testing.T) {
+	if testVersion != targetTestVersion {
+		t.Errorf("Found testVersion = %v, want %v.", testVersion, targetTestVersion)
```

Comment:
Our mistake for not testing that. I am considering asking for it in a separate commit, but realize that that is a bit paradoxical because it *could* break existing solutions. I don't want to overthink it, though I can't think about it right now. In a few hours.